twitch_api.py

```diff
@@ -66,15 +29,15 @@ class TwitchHandlerInterface(ABC):
     """
 
     @abstractmethod
-    def get_stream(self, username: str) -> Optional[TwitchStreamInterface]:
+    def get_streams(self, streamers: list) -> dict:
         """
-        Grabs an individual stream object if passed streamer is live
+        Gets a list of streams based on streamers passed
 
         Args:
-            username (str): The username to check
+            streamers (list): The streamers to check for
 
         Returns:
-            TwitchStreamInterface: If the stream is live, else none
+            list: A list of TwitchStreamInterfaces
```

Comment:
I removed the TwitchStreamInterface and just used a simple TwitchStream, this comment needs updating